src/Microsoft.VisualBasic/tests/Microsoft.VisualBasic.Tests.csproj

```diff
@@ -34,4 +34,7 @@
   <ItemGroup>
     <EmbeddedResource Include="Resources\$(AssemblyName).rd.xml" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\Microsoft.VisualBasic.vbproj" />
+  </ItemGroup>
```

Comment:
Why is a `<ProjectReference/>` needed now?

Reply:
I don't know I do but when I remove it I get 15 errors, VS says I need it
![image](https://user-images.githubusercontent.com/12789369/47685317-50f97000-db93-11e8-9f68-3ffc0f2d7674.png)

Reply:
I found the issue and have deleted the change, a file was move to the wrong project. I would not have touched this project.

Reply:
The `<ProjectReference/>` is still included. Please remove if not needed.

Reply:
It keeps reappearing and I don't know why but now when I remove it I get 50 errors including the ones above. It seems related to commits by Bruce Bower-Smith around 7/24/2018 that I got when I did a fetch. When I look at the file history there are definitely changes in addition to just the addition of the ProjectReference. I will research more tonight.

Reply:
I think this project reference should not be here either.